Source/WebKitLegacy/mac/WebView/WebFrame.mm

```diff
@@ -102,6 +102,7 @@
 #import <WebCore/PluginData.h>
 #import <WebCore/PrintContext.h>
 #import <WebCore/Range.h>
+#import <WebCore/RemoteInputEventData.h>
```

Comment:
Is this import needed for unified sources?

Reply:
It's needed for the `dragSourceEndedAt()` call in this file which now returns a `std::optional<WebCore::RemoteUserInputEventData>`.